src/App.scss

```diff
@@ -34,12 +34,36 @@ $fa-font-path: "~font-awesome/fonts";
   }
 }
 
+.btn.btn-outline-secondary{
+  border: 1px solid $deep-sea-blue;
+  color: $deep-sea-blue;
+
+  &:hover{
+    background-color: $pale-grey;
+    color: $deep-sea-blue;
+  }
+
+  &:focus{
+    background-color: $pale-grey;
+    color: $deep-sea-blue;
+  }
+
+  &:not(:disabled):not(.disabled):active{
+    background-color: $pale-grey;
+    color: $deep-sea-blue;
+  }
+
+  &.disabled{
+    border: 1px solid $slate-grey;
+    color: $slate-grey;
+  }
+}
 .btn.control-btn{
     line-height: 1;
     padding: 8px;
-    color: $primary;
```

Comment:
Is these styled buttons something we wouldn't want to change with the theme?

Reply:
ooo yeah good point - i'll put that back to primary and i'll make the next/prev buttons use primary color too